app/src/main/java/br/com/mob1st/bet/features/competitions/presentation/CompetitionsTabScreen.kt

```diff
@@ -1,49 +1,240 @@
 package br.com.mob1st.bet.features.competitions.presentation
 
-import androidx.compose.animation.Crossfade
+import androidx.activity.compose.BackHandler
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Home
 import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Divider
+import androidx.compose.material3.ExperimentalMaterial3Api
 import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
 import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SuggestionChip
+import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
+import androidx.compose.runtime.rememberUpdatedState
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
 import androidx.lifecycle.compose.ExperimentalLifecycleComposeApi
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import br.com.mob1st.bet.core.localization.getText
-import br.com.mob1st.bet.core.ui.ds.InfoTemplate
-import br.com.mob1st.bet.features.competitions.domain.CompetitionEntry
-import org.koin.androidx.compose.koinViewModel
-import org.koin.core.parameter.parametersOf
+import br.com.mob1st.bet.core.ui.compose.LocalLazyListState
+import br.com.mob1st.bet.core.ui.compose.LocalSnackbarState
+import br.com.mob1st.bet.core.ui.ds.atoms.CompositionLocalGrid
+import br.com.mob1st.bet.core.ui.ds.molecule.RetrySnackbar
+import br.com.mob1st.bet.core.ui.ds.organisms.FetchedCrossfade
+import br.com.mob1st.bet.core.ui.ds.page.DefaultErrorPage
+import br.com.mob1st.bet.core.ui.ds.templates.InfoTemplate
+import br.com.mob1st.bet.core.ui.state.AsyncState
+import br.com.mob1st.bet.core.ui.state.SimpleMessage
+import br.com.mob1st.bet.core.utils.extensions.ifNotEmpty
+import br.com.mob1st.bet.features.competitions.domain.IntScores
+import br.com.mob1st.bet.features.competitions.domain.MatchWinner
 
 @OptIn(ExperimentalLifecycleComposeApi::class)
 @Composable
-fun CompetitionsTabScreen(entry: CompetitionEntry) {
-    val viewModel = koinViewModel<ConfrontationListViewModel> {
-        parametersOf(entry)
+fun SelectedCompetition(
```

Comment:
É ok ter duas telas no mesmo arquivo?

Reply:
Desde que elas herdem o mesmo viewModel?

Reply:
Aah tu mudou no outro PR que abriu

Reply:
isso, to pensando tentando manter uma tela por arquivo.

minha ideia é manter uma organização que o arquivo da tela separe composable stateful de stateless. 
por exemplo, um app de padaria que tem uma tela que mostra um **pão**:
- PãoScreen: stateful, inicia o viewmodel e guarda o estado da tela e chama o PãoPage 
- PãoPage: desenha o pão, nao tem estado, nao sabe o que fazer com clicks, só recebe parametros e desenha as coisas

Então screen seria o que tem estado, faz navegações, cria viewmodels
E page seria só UI.
O código ficaria mais ou menos assim
```kotlin
@Composable
fun PaoScreen() {
    val paoViewModel = koinViewModel<PaoViewModel>()
    val state by paoViewModel.uiState.collectAsStateWithLifecycle()
    PaoPage(
        pao = state,
        onClick = { paoViewModel.fromUi(event = OnClick) }
    )
}

@Composable
fun PaoPage(
    state: AsyncState<PaoData>,
    onClick: () -> Unit,
) {
    // desenha o pao
}
```